.cirrus.yml

```diff
@@ -126,8 +126,8 @@ yjit_task:
   test_dump_insns_script: RUST_BACKTRACE=1 ./miniruby --yjit-call-threshold=1 --yjit-dump-insns -e0
   output_stats_script: RUST_BACKTRACE=1 ./miniruby --yjit-call-threshold=1 --yjit-stats -e0
   full_build_script: source $HOME/.cargo/env && make -j
-  make_rdoc_script: source $HOME/.cargo/env && make -j rdoc
-  make_btest_script: source $HOME/.cargo/env && make -j btest RUN_OPTS="--yjit-call-threshold=1 --yjit-verify-ctx"
+  cargo_test_script: source $HOME/.cargo/env && cd yjit && cargo test
+  make_btest_script: source $HOME/.cargo/env && make -j test RUN_OPTS="--yjit-call-threshold=1 --yjit-verify-ctx"
```

Comment:
Here we're already running `test-all` and `test-spec`

Is `make test` a superset of that?

Reply:
It's like:

* `make check`
  * `make test`
     * `make btest`
     * `make test-basic`
  * `make test-all`
  * `make test-spec`

I'm open to just using `make check` if we want to make YMAL shorter, but I feel like this level of separation makes it easier to see what's happening when it fails.

Reply:
Yes I agree, I prefer having things separated out as well 👍